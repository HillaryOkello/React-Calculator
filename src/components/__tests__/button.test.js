import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../button';

it('renders correctly', () => {
  const isWide = true;
  const tree = renderer
    .create(<Button name="0" key="0" clickHandler={() => {}} wide={isWide} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
