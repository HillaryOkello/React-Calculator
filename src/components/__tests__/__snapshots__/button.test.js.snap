// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  onClick={[Function]}
  style={
    Object {
      "background": "orange",
      "font": "3rem 'Syne Mono'",
      "height": "100%",
      "textAlign": "center",
      "width": "50%",
    }
  }
  type="button"
>
  0
</button>
`;
